### DISCUSSION TITLE
Is copilot no longer free for students?

### DISCUSSION THREAD

**Discussion by fedorkanin** [community/community#41371] [Copilot Conversations] [answered]
### Github Copilot

Question

Github Copilot was free for students and teachers with confirmed student status. Is it no longer the case?

Edit: issue is fixed
<img width="796" alt="image" src="https://user-images.githubusercontent.com/45014559/206880597-bef32f4a-fc54-4f86-99e3-31f76e0b1f31.png">

**Comment by Vrajs16** [comment]
I am having the same problem. I checked GitHub develop campus and saw that my email is still verified
![image](https://user-images.githubusercontent.com/89588024/206879341-08421f52-8161-43d8-a051-77cd69075cd8.png)

in visual studio code it's not working and getting this error: 
![image](https://user-images.githubusercontent.com/89588024/206879348-1fcb6bdc-29c8-43e8-8716-b60de10b6771.png)
I logged out and did everything yet nothing is working?

Edit:
Got an email saying it's renewed, but only for like a month?
![image](https://user-images.githubusercontent.com/89588024/206879437-0d784861-90fd-434b-a8fd-0a6d16f63539.png)

**Reply by Dan0xE** [reply]
they seem to check the eligibility monthly now, pretty sure that's the reason why you got this email 

![unnamed](https://user-images.githubusercontent.com/23116945/211236782-45ea1784-84b1-4944-b214-c2a852425d1d.jpg)

**Reply by Haidar0096** [reply]
how did you solve this?

> I am having the same problem. I checked GitHub develop campus and saw that my email is still verified ![image](https://user-images.githubusercontent.com/89588024/206879341-08421f52-8161-43d8-a051-77cd69075cd8.png)
> 
> in visual studio code it's not working and getting this error: ![image](https://user-images.githubusercontent.com/89588024/206879348-1fcb6bdc-29c8-43e8-8716-b60de10b6771.png) I logged out and did everything yet nothing is working?
> 
> Edit: Got an email saying it's renewed, but only for like a month? ![image](https://user-images.githubusercontent.com/89588024/206879437-0d784861-90fd-434b-a8fd-0a6d16f63539.png)

How did you manage to solve this? Ihave the exact same problem

**Comment by FGBxRamel** [comment]
I have the same question. I've got the exact problem mentioned in [this discussion](https://github.com/orgs/community/discussions/40019) .

**Comment by Iheuzio** [comment]
I have the same issue across all my ides, was fine a few hours ago now... 
![image](https://user-images.githubusercontent.com/97270760/206879371-a9e119ce-58ee-484c-aef2-d8d107f3a220.png)

**Reply by Iheuzio** [reply]
update: 
![image](https://user-images.githubusercontent.com/97270760/206880197-4378c162-aa38-450e-98d2-6711b894f50d.png)

**Comment by PaulFWatts** [comment]
Exactly the same problem here.

Hopefully it's just a glitch?

**Comment by savicaleksa** [comment]
Same problem here.

**Comment by Dradows** [comment]
Same here!

**Comment by velrmilion** [comment]
Having the same issue here.

**Comment by HDcookie** [comment]
I'm in the same boat, I can't use it at all, and no email

**Comment by K-cermak** [comment]
Same issue here.

**Comment by i13e** [comment]
I am having the same issue and just received ~~3 emails~~ now 4 emails from Github Support saying the exact same thing as the top comment.

**Comment by rafael-rm** [comment]
I'm having the same problem, I hope it's just an error.

**Comment by HDcookie** [comment]
Looks like it's fixed
<img width="404" alt="image" src="https://user-images.githubusercontent.com/98278469/206880317-f0b8a686-5bb9-4286-8c27-67ccd44143e4.png">
![image](https://user-images.githubusercontent.com/98278469/206880319-ad3cc6ff-1fb6-46ca-9b6b-188111dcf32b.png)

**Comment by blakeNaccarato** [comment]
### Solution that worked for me

Thanks to my GitHub support person for helping me sort this out. I visited the link below and was able to reactivate Copilot.

<https://github.com/github-copilot/free_signup>

(Note: This link doesn't work properly on the GitHub mobile app! Open it in your browser!)

So please try this solution before opening a ticket yourself, otherwise keep reading for my original comment.

____

### If you're still having issues after trying the above...

I am writing a meta-comment here to link a few **active** discussions about this issue:

- **https://github.com/orgs/community/discussions/41371** (this discussion, most votes)
- https://github.com/orgs/community/discussions/41378
- https://github.com/community/Global-Campus-Teachers/discussions/118#discussioncomment-4369255 (my comment on this issue)

And here are a few **apparently resolved/locked** discussions about a bad deploy causing this issue about a month ago:

- https://github.com/orgs/community/discussions/40025
- https://github.com/orgs/community/discussions/40037

The [dev response](https://github.com/orgs/community/discussions/40037#discussioncomment-4234429) implies this affected a small percentage of users, and the change was rolled back a month ago. Perhaps the change has rolled out again without a fix?

**Should we concentrate discussion here, the highest voted among the active issues? It specifically asks about Copilot for students, but it represents the teacher situation as well.**

A suggested workaround is to open a ticket, via this comment: https://github.com/community/Global-Campus-Teachers/discussions/118#discussioncomment-4371361

If the solution above *didn't* work for you, please leave a comment indicating that.

**Reply by detykoruqeho6** [reply]
The first free_signup URL worked for me,cop is working again

**Reply by Haidar0096** [reply]
this didn't work for me, any suggestions?

**Reply by umarhunter** [reply]
I can't believe this was the solution all along!

**Reply by jieunboy0516** [reply]
thanks worked for me.

**Reply by Nunomst** [reply]
Thanks!

**Reply by Alukardie** [reply]
Finally, thanks!!

**Reply by im-ThiagoC** [reply]
Thanks!

**Reply by HungChunLi** [reply]
Thanks!

**Comment by FDD-YOSA** [comment]
To this day there is still this problem. They are so slow to fix it.

**Reply by blakeNaccarato** [reply]
Just checking in, have you tried going to the following link and seeing if the option is there for you to sign up for free? Heads up, this link acts strangely in the GitHub mobile app, I believe it tries to process it as a repository link. So open it in your browser instead.

<https://github.com/github-copilot/free_signup>

**Reply by avivajpeyi** [reply]
The https://github.com/github-copilot/free_signup link takes me to https://github.com/github-copilot/signup:

<img width="552" alt="Screen Shot 2023-01-09 at 11 23 17 am" src="https://user-images.githubusercontent.com/15642823/211226342-e3713fc8-6ad0-4566-8cfd-763235b97393.png">

**Reply by akbaraffaruk** [reply]
same issue with me

![image](https://user-images.githubusercontent.com/65720654/211230741-8cfed739-3299-40d8-a580-1b833604a1bd.png)

**Reply by 3chospirits** [reply]
likewise!

**Reply by timelic** [reply]
same issue

**Reply by xiaobanni** [reply]
same issue with me!

**Reply by c19y9x** [reply]
same

**Reply by multipletwigs** [reply]
It was fine a few hours ago, and I got this issue like 10 minutes ago!

**Comment by otkrickey** [comment]
same issue on 9/Jan/2023

**Comment by raipen** [comment]
I have same issue on 9/Jan/2023

**Comment by Sy0307** [comment]
same issue on 9/Jan/2023

**Comment by tsukumijima** [comment]
I'm also having the exact same issue on Jan 9, 2023.
I've tried signing out of GitHub and signing back in, no change. I tried the Nightly version, but it had no effect.
I think this is clearly a GitHub bug.

**Comment by MisakaRehana** [comment]
same issue on 9/Jan/2023

**Comment by HikaruChang** [comment]
I had the same problem. It stopped working just now. But my student certification has not expired.

![DB675F5D-ACB5-4F43-8653-BC6D83623935](https://user-images.githubusercontent.com/16264281/211208919-400685e8-bd11-4767-be2a-d1567c9520aa.png)
![E4977FB2-8967-4A7F-BCDC-017B1C9F4C4C](https://user-images.githubusercontent.com/16264281/211208914-d760cc82-7b18-4175-966c-ec3f4d389373.png)

**Reply by HikaruChang** [reply]
I received an email, which has only been renewed for one month. :(

![D7EA603E-DB84-4D64-BD24-87FCCE09E292](https://user-images.githubusercontent.com/16264281/211234033-5b954203-f4ca-4664-9eba-e473b592fa8a.jpeg)
![9A287E55-1E72-4398-8988-BEC41A57C2F3](https://user-images.githubusercontent.com/16264281/211234038-9b8aa676-0259-4834-b4ed-0a46e13d45b0.jpeg)

**Reply by HikaruChang** [reply]
I seem to have found the answer.
![17D26F6C-8531-4D4E-B689-3B48B323FF04](https://user-images.githubusercontent.com/16264281/211235513-57d393d7-064c-4986-b48c-c9bed1e2629f.jpeg)

**Reply by Haidar0096** [reply]
@hikaruchang how did you manage to solve it? I have the same problem today?

**Comment by ryhn7** [comment]
Same issue here on 9/Jan/2023, i'm still have verified student account
![WhatsApp Image 2023-01-09 at 1 56 31 AM](https://user-images.githubusercontent.com/72327884/211213765-0dbc5b37-290d-43e3-b264-8d241da2eafc.jpeg)

**Comment by otkrickey** [comment]
I have created a discussion for today's case. Please share your situation. #43634

**Comment by blakeNaccarato** [comment]
Here's a repost of my comment above for the wave of new subscribers to this thread. Does the link below work for you? It only works properly *outside of the mobile app*.

### Solution that worked for me

Thanks to my GitHub support person for helping me sort this out. I visited the link below and was able to reactivate Copilot.

<https://github.com/github-copilot/free_signup>

(Note: This link doesn't work properly on the GitHub mobile app! Open it in your browser!)

So please try this solution before opening a ticket yourself, otherwise keep reading for my original comment.

**Reply by JasinRefiku** [reply]
Nope, it won't work. The subscription page will still shows up.

**Reply by KhaldiDataPanda** [reply]
> Nope, it won't work. The subscription page will still show up.
same here nthn changed

**Reply by tsukumijima** [reply]
Unfortunately, it didn't work for me. I'm always redirected to https://github.com/github-copilot/signup.

**Reply by blakeNaccarato** [reply]
Yeah, it seems my Copilot phased out again as well

**Reply by yxistz** [reply]
Even when opened on browser

**Comment by KhaldiDataPanda** [comment]
The problem hits again , same error in vs code and this page 

![Screenshot 2023-01-09 012623](https://user-images.githubusercontent.com/98955624/211226331-a951beac-ce86-44e3-98d1-befe1e3b9857.png)

**Comment by Atepir** [comment]
same issue on jan 9

**Comment by eissa220** [comment]
same issue on jan 9

**Comment by vqthanhdev** [comment]
same issue for me. 09:25(+7) 9 Jan. How to fix it. My student account is not expired now.

**Comment by ThreeCatsLoveFish** [comment]
same issue on jan 9

**Comment by leijuns** [comment]
same issue on jan 9

**Comment by rikurunico** [comment]
I have same issue on 9/Jan/2023

**Comment by ZhengzheXu** [comment]
same issue on jan 9

**Comment by inannan423** [comment]
same issue on jan 9,that's very frustrating for me

**Comment by yoke0104x** [comment]
same issue on jan 9,that's very frustrating for me

**Comment by Dan0xE** [comment]
same issue here, I assume its due to policy changes (they seem to check the eligibility monthly now, that's why some people seem to get the email saying that their subscription has been extended by one month )

![unnamed](https://user-images.githubusercontent.com/23116945/211236581-05b4da69-a3f1-45dd-8d30-10ce8b496826.jpg)

(Credits to Hikaru Chang)

**Reply by HikaruChang** [reply]
I think GitHub will check every month to see if you are eligible to continue using it for free.

**Comment by Jm-Paunlagui** [comment]
Received this email too?
![image](https://user-images.githubusercontent.com/95494414/211235846-6a45625c-406a-4820-84d3-1d55995b304f.png)
But verified student
![image](https://user-images.githubusercontent.com/95494414/211235921-7699caf1-4af5-4bac-a590-1d460c6a0407.png)
Haiiiyaa,

**Comment by aadityamayankar** [comment]
Facing the same issue as a lot of others.

> I seem to have found the answer. ![17D26F6C-8531-4D4E-B689-3B48B323FF04](https://user-images.githubusercontent.com/16264281/211235513-57d393d7-064c-4986-b48c-c9bed1e2629f.jpeg)

Does this :point_up: mean we'll have to open a ticket every month?

**Reply by HikaruChang** [reply]
I think GitHub will check every month to see if you are eligible to continue using it for free.

**Comment by 343N** [comment]
Same issue here. Any solutions? Previously suggested solutions don't work, I get redirected to buy Copilot. I still have GitHub Pro though.

**Comment by otkrickey** [comment]
Solved: #43634

**Reply by otkrickey** [reply]
@xiaobanni

**Reply by xiaobanni** [reply]
Thanks for your careful analysis.

**Comment by marcosncosta1** [comment]
Did anyone find a solution? I am having the same problem here

**Comment by savicaleksa** [comment]
From time to time, Github will check if you're still a valid student and if you're allowed to use Github Copilot for free.
Don't worry, it's not a long process, and you'll get your Github Copilot back in no time.

**Reply by ff781** [reply]
So I just signed up 2 days ago, do I just have to wait?

If I go to https://github.com/github-copilot/signup it just asks me to buy the paid plan

**Comment by Eddie2111** [comment]
Alright, same issue again

![image](https://github.com/community/community/assets/59513101/48f38d9f-a088-4057-b300-66ea05a0ef29)

My account is still a verified student account. Will be renewed again right?

**Reply by davecheney** [reply, marked as answer]
@Eddie2111 your student coupon has expired. If you are a current student you can reapply for the GitHub student pack and you will receive a coupon for an additional 2 years of copilot access

**Comment by aslamanver** [comment]
Same issue

**Comment by Tharuka-Herath** [comment]
<img width="1288" height="666" alt="image" src="https://github.com/user-attachments/assets/d9666488-cab7-4b6b-804a-c0687585b18b" />
<img width="1627" height="778" alt="image" src="https://github.com/user-attachments/assets/ea7dd15d-ba1b-4ee9-9c07-a05a5181e81a" />

I got the access to github student pack today but copilot is still on free plan. how to fix this

**Reply by ggaaddddyy** [reply]
same problem [Tharuka-Herath](https://github.com/Tharuka-Herath).  still on free plan despite student verification

**Reply by ggaaddddyy** [reply]
Oh wait, they just paused sign-ups indefinitely..... why did I even go to the trouble to sign up lol 

"Student sign-ups for Copilot are currently paused
Your frustration makes sense — this pause affects new student activations even if you completed the application.

Here’s what’s accurate based on the current GitHub documentation:

Yes, new student access to GitHub Copilot is temporarily paused.

According to the official docs, starting April 20, 2026, GitHub paused new sign-ups for:

Copilot Student
Copilot Pro
Copilot Pro+
This means that even if you recently applied for GitHub Education or completed the student verification steps, you won’t be able to activate Copilot Student right now if you didn’t already have access before the pause.

A few important clarifications:

This is described as a temporary pause, not an announced permanent or indefinite shutdown.
Existing student Copilot access continues to work without interruption.
GitHub hasn’t published a date for when new student activations will reopen.
You can confirm this directly in the docs under Access GitHub Copilot for free as a student, which includes the pause notice: [https://docs.github.com/en/copilot/how-tos/copilot-on-github/set-up-copilot/enable-copilot/set-up-for-students (opens in a new tab)](https://docs.github.com/en/copilot/how-tos/copilot-on-github/set-up-copilot/enable-copilot/set-up-for-students)

What you can do right now
You can still use Copilot Free, which is available to most individual accounts with limited usage.
If Copilot Student access reopens in the future, having your GitHub Education verification completed should let you activate it quickly at that time.
If this pause blocks something critical for you and you need an official confirmation tied to your account or eligibility status, go ahead and submit the support ticket — GitHub Support can check account-specific details that aren’t visible here."

**Comment by iamaanahmad** [comment]
New limits made the Github Copilot experience worst for Students Pack, very limited credits and can't even use in Github web, or writing commit message on VS Code. Github made it very CHEAP experience.

Disappointed!
